Make solvers of the cluster issuers configurable (#8)
projectsyn/component-cert-manager: docs/modules/ROOT/pages/references/parameters.adoc

The version of the Helm chart to use.

== `letsencrypt_email`

[horizontal]
type:: string
default:: ``

> The email Let's Encrypt will use to contact you.

Reviewer: ```suggestion
The email address Let's Encrypt will use to contact you.
```

default:: ``

The value of `no_proxy` is passed to cert-manager in environment variable `NO_PROXY`.

== `solvers`

[horizontal]
type:: array
default:: see `defaults.yml`

> An array holdig the solvers for the default cluster issuers.
> 
> See https://cert-manager.io/docs/configuration/acme/#configuration for posible values.

Reviewer: ```suggestion
An dictionary holding the solvers for the default cluster issuers.

See https://cert-manager.io/docs/configuration/acme/#configuration for configurable parameters.
```

== Example

[source,yaml]
----
solvers:

>   - http01:
>       ingress:
>         podTemplate:
>           metadata:
>             labels:
>               app: "solver"

Reviewer: ```suggestion
  http01:
    ingress:
      podTemplate:
        metadata:
          labels:
            app: "solver"
```
